python/hugr_qir/hugr_to_qir.py

```diff
@@ -0,0 +1,18 @@
+import tempfile
+from pathlib import Path
+
+from .cli import hugr_qir_impl
+
+
+def hugr_to_qir(hugr: bytes, validate_qir: bool = True) -> str:
+    """A function for converting hugr to qir."""
```

Comment:
Could you add some details in the doc string here about the parameters? For example which type of hugr you are expecting here?

Reply:
(and the return type)